src/SimConnectConnection.ts

```diff
@@ -168,6 +172,10 @@ class SimConnectConnection extends EventEmitter {
                 this.emit('exception', new RecvException(data));
                 break;
             case RecvID.ID_OPEN:
+                if(this.openTimeout >= 0) {
+                    clearTimeout(this.openTimeout);
```

Comment:
I'm trying to think of other cases where we want to clear the timeout. Perhaps `this.clientSocket.on('error')`. Are there any other? 🤔

Reply:
Timeout is only set when successful connection was made, so after this.clientSocket 'connect' event is fired. I'd however consider adding clearTimeout in SimConnectConnection.close() just in case.

Reply:
Ahh you’re right, no need to do it in the error handler then. But adding it in SimConnectConnection.close() sounds like a good idea, in case someone wants to cancel the connection manually.

Reply:
Added this